rollup.config.js

```diff
@@ -81,8 +81,9 @@ function createCommonJSConfig(input, output, options) {
     output: {
       file: `${output}.js`,
       format: 'cjs',
+      esModule: false,
       outro: options.addModuleExport
-        ? 'module.exports = exports.default; Object.assign(exports.default, exports);'
+        ? ';(module.exports = (exports && exports.default) || {}),\n  Object.assign(module.exports, exports)'
```

Comment:
why `,\n` instead of `;`? Mostly out of curiosity.

Reply:
I wrote it in a minified bundle locally and copied it from there so that’s the minified version

Reply:
I'm not convinced that `,\n` is more minified than `;`, but don't care much..

Reply:
Yeah, I didn't do it 😂, it was from the minifier of the I was trying all this with and most of them do that 
<img width="1019" alt="Screenshot 2022-12-17 at 5 52 29 PM" src="https://user-images.githubusercontent.com/43572006/208241639-93e50714-9e29-4eca-9cda-70526ced92eb.png">

Reply:
that's actually minified. but adding `;(...)` increased 3 bytes.

What I would do is something like:
```suggestion
        ? ';module.exports=exports&&exports.default||{};Object.assign(module.exports,exports)'
```

Hmm, we can actually combine it?
```suggestion
        ? ';Object.assign((module.exports=exports&&exports.default||{}),exports)'
```

(I'm just enjoying the puzzle. Whatever works is okay to merge.)

Reply:
That'll need another addition of `,` cause the return of the above would be either exports or an empty object and not actually assign stuff to `exports` so 
`Object.assign(((module.exports = (exports && exports.default) || {}), module.exports),exports);`

Would be the working one and compared to that the original commit code is smaller 😂